README.md

```diff
@@ -51,10 +50,12 @@ On top of that, AEM developers can extend the AEM Search Component to implement
 out-of-the-box. The component follows the architectural patterns of [AEM WCM Core Components](https://github.com/adobe/aem-core-wcm-components) . Therefore, the approach is
 known and straight forward to extend while ensuring maintainability of the core module. In the SAAS Administration UI
 the SAAS Crawler can be configured. A recommended approach is to use Sitemaps (although raw crawling works as well) in
-order to indicate which pages should be shown in the Search Admin. To generate Sitemaps, the Apache Sling Sitemap module
+order to indicate which pages should be shown. To generate Sitemaps, the Apache Sling Sitemap module
 can be used described on the AEM
 documentation: https://experienceleague.adobe.com/docs/experience-manager-cloud-service/overview/seo-and-url-management.html?lang=en#building-an-xml-sitemap-on-aem
 
+Metadata such as a title, description, publication date, taxonomy and more can be scrapped from the HTML or Sitemaps in order to allow the usage of advanced features such as facetting, filtering or customized sorting.   
```

Comment:
.. can be _scraped_ ... (1 p, not 2, because it's from the verb "to scrape", not "to scrap")

Also not sure if the word "facetting" actually exists, maybe better say something like "to use facets"? But not sure if this is a good solution either.